ruby/build.gradle

```diff
@@ -20,9 +20,9 @@ task buildSempLib << {
     def f = new File('../swagger.yaml')
     if (!f.exists()) {
         new URL('https://sftp.solace.com/download/VMR_SEMPV2_SCHEMA_YAML').withInputStream{ i -> f.withOutputStream{ it << i }}
```

Comment:
I think this link also needs an update.

Reply:
Huh, I was sure I fixed both `ruby` and `python`. In any case I've pushed an update.